ibm/resource_ibm_iam_api_key.go

```diff
@@ -170,44 +179,41 @@ func resourceIbmIamApiKeyRead(context context.Context, d *schema.ResourceData, m
 		return diag.FromErr(err)
 	}
 
```

Comment:
In read method here in below code add apikey == nil condition also
apiKey, response, err := iamIdentityClient.GetAPIKey(getApiKeyOptions)
	if err != nil || apikey == nil {